### ISSUE TITLE
summary.spdur for models without risk intercept

### ISSUE THREAD

**Issue by andybega** [andybega/spduration#16]
Currently `summary.spdur` uses the location of the risk intercept to split the data frame of coefficients into separate pieces for the duration and risk equation:

from `summary.spdur.R`:

```
start_split <- which(names(object$coefficients)=='(Risk Intercept)')
end_duration <- start_split - 1
```

So if there is no risk intercept, i.e. a model with `atrisk ~ -1 + ...`, these variables will be missing and will lead to an error. 

Maybe ID split by `attr(model$mf.dur, "terms") %>% attr(., "intercept")` in combination with number of terms.

**Comment by andybega** [comment]
Partial fix that doesn't work with models that have no risk intercept but factor or character variables in the duration equation. See #17

**Comment by andybega** [comment]
Fixed in commit that fixes #17
